.env.example

```
@@ -1,6 +1,6 @@
 # Database connection settings
-MARIADB__HOST=
-MARIADB__PORT=
+MARIADB__HOSTNAME=
+MARIADB__EXPOSE__PORT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MARIADB__EXPOSE__PORT key should go before the MARIADB__HOSTNAME key

(UnorderedKey)